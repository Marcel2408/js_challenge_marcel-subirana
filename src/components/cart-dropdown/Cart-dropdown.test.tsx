import React from 'react';
import renderer from 'react-test-renderer';
import CartDropdown from './Cart-dropdown.component';

it('expect to render Cart-dropdown component', () => {
  const actualSnapshot = renderer.create(<CartDropdown />).toJSON();
  expect(actualSnapshot).toMatchSnapshot();
});
