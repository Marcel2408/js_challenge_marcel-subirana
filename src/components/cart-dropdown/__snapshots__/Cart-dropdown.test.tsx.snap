// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Cart-dropdown component 1`] = `
<div
  className="cart-dropdown"
  onMouseLeave={[Function]}
>
  <div
    className="cart-items"
  >
    <span
      className="empty-message"
    >
      Your cart is empty
    </span>
  </div>
  <button
    className="button button--primary button--checkout"
    type="button"
  >
    Go to checkout
  </button>
</div>
`;
